pkg/api/types.go

```diff
@@ -153,6 +153,15 @@ const (
 	TerminationMessagePathDefault string = "/dev/termination-log"
 )
 
+// There is no namespace required for these kinds specified here.
+// If a kind is not enumerated here, it is assumed to have a namespace scope.
+var NoneNamespacedKinds = []string{
```

Comment:
The disadvantage of this is that the list is now far away from the place where types are actually installed, which means it will probably get forgotten.

@lavalamp @smarterclayton @derekwaynecarr for ideas on how to do this best